metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -85,6 +97,11 @@ object MetalsServerConfig {
       case "false" | "off" => false
       case _ => default
     }
+  def intOption(key: String, default: Int): Int =
```

Comment:
I can see it is similar to `binaryOption` but it is not a good idea to silently ignore user properties if they are not parsed correctly. 
I would at least log an error/warning that parameter could not be parsed and default value is used.

Reply:
Fixed, added log messages for binary and int options.